src/components/Home/HeroSection.js

```diff
@@ -1,20 +1,31 @@
 import React from 'react'
 import video from '../../assets/videos/anime.mp4'
 import "../../styles/components/common/HeroSection.css"
-import { Link } from 'react-router-dom'
+import { Link as ScrollLink } from 'react-scroll';
 const HeroSection = () => {
   return (
     <div className='hero-container'>
       <video src={video} autoPlay muted loop />
       <h1>Welcome to <span className="text-primary">Pec</span> Acm</h1>
       <p>Lets have some Fun!!</p>
       <div className='hero-btns'>
-        <Link to='/games'> {/* Link to the '/games' route */}
-          <button type="button" className="btn btn-primary">Games</button>
-        </Link>
-        <Link to='/activities'> {/* Link to the '/activities' route */}
-          <button type="button" className="btn btn-light">Activities</button>
-        </Link>      </div>
+        <ScrollLink
+          to="games-section" 
+          smooth={true}
+          duration={500}
+          className="btn btn-primary"
+        >
+          Games
+        </ScrollLink>
+        <ScrollLink
+          to="activities-section" 
+          smooth={true}
+          duration={500}
+          className="btn btn-light"
+        >
+          Activities
+        </ScrollLink>
```

Comment:
This button is currently doing nothing cuz games and activities are shown together below
You can either direct it to `games-section` as well or separate the cards below into games and activities and link accordingly